packages/vue/src/components/sign-in.vue

```diff
@@ -22,22 +28,27 @@ const forgotYourPasswordLink = computed(() =>
 );
 
 const signInButtonText = computed(() => translate('Sign in'));
-const signIngButtonText = computed(() => translate('Signing in'));
 
-const { state, send } = useAuth();
-const actorState = computed(() =>
-  getActorState(state.value)
-) as ComputedRef<SignInState>;
+const { error, isPending, state, send, updateForm } = useAuthenticator();
```

Comment:
@ErikCH I'm struggling with the reactivity of `error` and `isPending` for the following examples:

- http://localhost:3000/ui/components/authenticator/sign-in-with-phone
- http://localhost:3000/ui/components/authenticator/sign-in-with-email
- http://localhost:3000/ui/components/authenticator/sign-in-with-username

Reply:
👀

Reply:
Ok, I fixed reactivity and also made some tweaks to sign-in

Reply:
Thank you @ErikCH !